src/shared/utilities/editorUtilities.ts

```diff
@@ -15,3 +16,35 @@ export function getTabSizeSetting(): number {
 export function getInlineSuggestEnabled(): boolean {
     return Settings.instance.getSection('editor').get('inlineSuggest.enabled', true)
 }
+
+export async function getOpenFilesInWindow(
```

Comment:
```suggestion
// Unlike vscode.window.visibleTextEditors, this function is needed because ...
// 
// See also: https://github.com/microsoft/vscode/issues/8886#issuecomment-259158438
export async function getOpenFilesInWindow(
```